### DISCUSSION TITLE
Can `limit` be used inside of `Include` when using the backend API?

### DISCUSSION THREAD

**Discussion by coderreco** [swellstores/community#413] [Help] [answered]
When using include in a call to the backend-api, can I use limit the results of that call? 

Example:
```
// get account data from swell database using backend api
const accountData = await swellnode.get('/accounts/{id}', {
   "id": accountId,
  "include": {
        "latest_cart": {
            "url": "/carts",
            "params": {
                "account_id": "id"
            },
            "data": {
                "abandoned": true,
                "sort": "date_last_accessed desc"
            }
        }
    }
}
```

Can I limit this to just 1 cart?

Thanks!

—Asked this in comments of another thread but deleted and moved to here

**Comment by ericingram** [comment, marked as answer]
You can use a URL like “/carts/:last” to have it return a single record as the result

The “:last” segment refers to the latest record depending on how the collection is sorted, and by default it’s sorted by date, hence it refers to the latest cart

Alternatively you can add “limit: 1” to the data object,  it then it would return a paginated result, so I imagine you would prefer the former approach

**Reply by coderreco** [reply]
Amazing. `:last` is even better. Thanks Eric.
